library/src/main/java/com/github/pwittchen/prefser/library/PreferencesAccessorsProvider.java

```diff
@@ -48,7 +48,7 @@ private void createAccessors() {
   private void createBooleanAccessor() {
     accessors.put(Boolean.class, new Accessor<Boolean>() {
       @Override public Boolean get(String key, Boolean defaultValue) {
-        return preferences.getBoolean(key, defaultValue);
+        return preferences.getBoolean(key, (defaultValue == null) ? false : defaultValue);
```

Comment:
No, we are not defining default behavior for the user, because if the user sends null in defValue, it would already return before in Prefser class (and it would return null).